packages/pointer_interceptor/pointer_interceptor_platform_interface/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: pointer_interceptor_platform_interface
+description: "A common platform interface for the pointer_interceptor plugin."
+version: 1.0.0
+publish_to: none
```

Comment:
This needs to be removed. That will then trigger more checks in CI; you'll need things like the issues and repository links that we have in all the pubspecs.